src/main/java/org/amoseman/steganography/Main.java

```diff
@@ -0,0 +1,10 @@
+package org.amoseman.steganography;
+
+import org.amoseman.steganography.ui.command.Parameters;
+import org.amoseman.steganography.ui.command.UserInterface;
+
+public class Main {
+    public static void main(String[] args) {
+        Parameters parameters = new UserInterface().run(args);
```

Comment:
Just a small thing. The method _run_ is a part of the Runnable interface so I try to avoid it as it looks like it is a threaded execution.  This is just a preference, not a critical issue.

Reply:
I've renamed run to parse.